rllib/algorithms/simple_q/simple_q.py

```diff
@@ -276,7 +276,12 @@ def validate(self) -> None:
         # Call super's validation method.
         super().validate()
 
-        if self.exploration_config["type"] == "ParameterNoise":
+        # TODO (simon): Find a clean solution to deal with
```

Comment:
Just a not: We were going to move SimpleQ into rllib_contrib, but never did b/c of some remaining dependencies. But we will not move this one into the new stack anyways.

Reply:
Alright - I was just referring to the problem with the new stack when exploration does take place via a certain exploration strategy and not via an action distribution. In DQN we have to deal with this to get somehow the `epsilon` into the `RLModule`